### ISSUE TITLE
TERRA-102 ⁃ Update docs with example for secure_bundles.url  (v2.1.15)

### ISSUE THREAD

**Issue by tringuyen-yw** [datastax/terraform-provider-astra#218]
Hi there,

Using **datastax/astra provider v2.1.15** The `secure_bundles.url` attribute is not accessible, contrary to the official documentation. See Reference section at the bottom of the ticket.

### Terraform Version
```
Terraform v1.4.2
on linux_amd64
+ provider registry.terraform.io/datastax/astra v2.1.15
```

### Affected Resource(s)
Please list the resources as a list, for example:
- data.astra_secure_connect_bundle_url

### Terraform Configuration Files
```hcl
data "astra_secure_connect_bundle_url" "securebundle-dev" {
  database_id = astra_database.astradb-dev.id
}

# this code is NOT working with astra provider v2.1.15
output "astra_securebundle_dev" {
  value = data.astra_secure_connect_bundle_url.securebundle-dev.secure_bundles.url
}
```

### Expected Behavior
Should return the strring of the URL to download the bundle. Please note the below code is OK OK with astra provider v2.1.14

```hcl
# WORKING OK with astra provider v2.1.14 (NOTE: not having secure_bundles attribute)
output "astra_securebundle_dev" {
  value = data.astra_secure_connect_bundle_url.securebundle-dev.url
}
```

### Actual Behavior
`terraform plan` failed by

```
╷
│ Error: Unsupported attribute
│ 
│   on outputs.tf line 2, in output "astra_securebundle_dev":
│    2:   value = data.astra_secure_connect_bundle_url.securebundle-dev.secure_bundles.url
│ 
│ Can't access attributes on a list of objects. Did you mean to access attribute "url" for a specific element of the list, or across all elements of the list?
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

### References
Documentation [astra_secure_connect_bundle_url (Data Source)](https://registry.terraform.io/providers/datastax/astra/latest/docs/data-sources/secure_connect_bundle_url#schema) states that

the [secure_bundles](https://registry.terraform.io/providers/datastax/astra/latest/docs/data-sources/secure_connect_bundle_url#nested-schema-for-secure_bundles) has a nested property `url` String type.

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-102) by [Unito](https://www.unito.io)
┆Issue Number: TERRA-102
┆Priority: Major

**Comment by emerkle826** [comment]
@tringuyen-yw Sorry about the typo, you are correct that `secure_bundles` needs to be there.

As for  the docs, they are technically correct as they `secure_bundles` as a `List`. But the examples on the page do not show how to reference the new datasource structure, only how to configure one. I'll get the docs updated as part of this ticket on the next release as the docs only get updated on releases.

**Comment by JanGe** [comment]
Our Terraform builds today broke too due to this unannounced breaking change.

@emerkle826 Could the Datastax team please make sure to **not introduce breaking changes** like these in patch version upgrades of the provider? The correct module version should have been 3.0.0 to make this obvious. Thank you!

**Comment by emerkle826** [comment]
@JanGe Sorry about that. I'll be more careful with breaking changes in the future. There will be another one coming soon to fix #170, which I will be updating the major version for most likely.
